llvm/lib/Target/AMDGPU/SILowerWWMCopies.cpp

```diff
@@ -137,5 +137,10 @@ bool SILowerWWMCopies::runOnMachineFunction(MachineFunction &MF) {
     }
   }
 
+  // SGPRs reserved for exec copy should be preserved as we encountered WWM_COPY
+  // instances.
+  if (Changed)
+    MFI->setPreserveExecCopyReservedReg();
```

Comment:
Do you really need to track a separate field for this? Can't you just check if RegForExecCopy is used?